donor-registry/src/app/layouts/layouts.component.ts

```diff
@@ -631,19 +632,22 @@ export class LayoutsComponent implements OnInit, OnChanges {
     })).subscribe((result: any) => {
     });
   }
-
-  downloadPledgeCard(index:any){
+ 
+  downloadPledgeCard(language: string,index:any){
     this.mode = this.getDeviceInfo();
     this.orientation = (!this.mode) ? "_landscape" : '_portrait';
-    let state = this.model[index]['addressDetails'].state;
-    state = state.replace(/ /g, "_");
-    this.documentName = state + this.orientation;
-
+    // let state = this.model[index]['addressDetails'].state;
+    // state = state.replace(/ /g, "_");
+    // this.documentName = state + this.orientation;
```

Comment:
@suvarnakale if this 'this.documentName' variable you are using anywhere remove it from declaration too

Reply:
okay